service/registrationservice.php

```diff
@@ -158,6 +163,9 @@ public function validateEmail($email) {
 		try {
 			return $this->registrationMapper->find($email);//if not found DB will throw a exception
 		} catch (DoesNotExistException $e) {}
+		catch (MultipleObjectsReturnedException $e) {
+			$this->registrationMapper->deleteAll($email);
```

Comment:
Automatically deleting all objects with same email upon finding multiple entries doesn't seem to be a good idea, since it shouldn't happen in the first place. My original intent was to let user report the error and let the administrator investigate. But I also see the importance of usability here, so please just add a single line to log the error as well.